apps/acrs/translations/src/en/pages.json

```diff
@@ -1,23 +1,54 @@
 {
+  "start": {
+    "header": "Continue to your form"
+  },
+  "information-you-have-given-us": {
+    "header": "Information you have given us"
+  },
   "confirm": {
     "header": "Check your answers",
     "sections": {
-      "name": {
-        "header": "Details"
+      "referral-details": {
+        "header": "Referral details"
+      },
+      "immigration-adviser-details": {
+        "header": "Immigration adviser details"
       }
     },
     "fields": {
-      "name": {
-        "label": "Project Name"
+      "brp": {
+       "label": "BRP"
+      },
+      "uan": {
+        "label": "UAN"
+      },
+      "date-of-birth": {
+        "label": "Date of birth"
+      },
+      "who-is-completing-form": {
+        "label": "Person completing form"
+      },
+      "full-name": {
+        "label": "Full name"
+      },
+      "confirm-referrer-email": {
+        "": "Email"
```

Comment:
Need 'label' here

Reply:
Probably need to check with UCD team about which fields should be in the summary and what the labels should be. For example we have 3 labeled 'Email' here. That's going to be confusing for users until we have some better advice.